config/locales/de.application_letters.yml

```diff
@@ -73,3 +73,11 @@ de:
     pending_before_deadline: "Beworben"
     pending_after_deadline: "In Bearbeitung"
     alternative: "Nachrücker"
+
+
+  controllers:
+    application_letters:
+      confirmation_mail:
+        sender: "workshop@hpi.de"
```

Comment:
I would have liked to see the usage of the default value or any other user changeable email here